eval/scenarios/self-review.md

```
@@ -0,0 +1,184 @@
+
+### TC-02: 問題のない変更差分のレビュー
+
+- **前提**:
+  - main ブランチから `feature/test-self-review-clean` ブランチを作成済み
+  - コーディング規約に準拠し、テストが充足した適切なコード変更をコミット済み
+
+- **入力**:
+  ```
+  /self-review
+  ```
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)